src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action in actions) {
+    if (actions[action].type === 'addProperties') {
+      for (const prop in actions[action]['extraData']) {
```

Comment:
```suggestion
      for (const prop in actions[action].extraData) {
```
Используй обращение через точку